### PR TITLE
Fix metastatus -vv

### LINKED ISSUE #257: error running metastatus -vv
```
robj@dev22-devc:~/yelpsoa-configs/paasta_canary (master) $ paasta metastatus -vv -c norcal-stagef
Cluster: norcal-stagef
Dashboards:
  Marathon RO: xxx
  Smartstack:  xx
  Chronos RO:  xx
  Signalfx:    xx
  Mesos:     xx

Traceback (most recent call last):
  File "/usr/bin/paasta_metastatus", line 460, in <module>
    main()
  File "/usr/bin/paasta_metastatus", line 411, in main
    mesos_results = get_mesos_status(mesos_state, verbosity=args.verbose)
  File "/usr/bin/paasta_metastatus", line 289, in get_mesos_status
    metrics_results.extend(run_healthchecks_with_param(mesos_state, [assert_extra_attribute_data]))
  File "/usr/bin/paasta_metastatus", line 297, in run_healthchecks_with_param
    return [healthcheck(param) for healthcheck in healthcheck_functions]
  File "/usr/bin/paasta_metastatus", line 261, in assert_extra_attribute_data
    '%.2f' % resources_remaining['free_resources']['disk'],
TypeError: 'int' object has no attribute '__getitem__'
```

note, it works with `-vvv` and `-v`

### PR DESCRIPTION
**Pull request by nhandler**
Not sure if this is enough to fix the issue with `paasta metastatus -vv` (it is tough to test this due to the nature of status/metastatus connecting to remote hosts). But I can iterate as necessary. 

Fixes #257

### CODE REVIEW

**Comment by solarkennedy** [comment]
Ship. I guess we have no test coverage for this function? Can you add some by pulling in some fake state data?

**Comment by nhandler** [comment]
Yeah. I have some partial work in my local checkout. I just need to figure out how to properly handle the 'fake state data' part.

**Comment by Rob-Johnson** [comment]
tests + shipit

File: paasta_itests/paasta_metastatus.feature [hunk 1]
```diff
@@ -52,8 +52,11 @@ Feature: paasta_metastatus describes the state of the paasta cluster
 #     When all masters are unavailable
 #     Then metastatus returns 2
 
-  Scenario:
+  Scenario: paasta metastatus verbose succeeds
     Given a working paasta cluster
      When we create a trivial marathon app
-     Then paasta_metastatus -vvv exits with return code "0" and outputs the slave's hostname
+     Then paasta_metastatus -v exits with return code "0" and output " "
```
Comment (Rob-Johnson) [inline paasta_itests/paasta_metastatus.feature]: what's going on here?

Comment (nhandler) [inline paasta_itests/paasta_metastatus.feature]: We had a scenario testing out `paasta metastatus -vvv`, but we didn't have any for `-v` or `-vv`. This just adds a bit more coverage to try and catch problems.

**Comment by mjksmith** [comment]
LGTM

**Pull request merged**